ブランチを作成してプッシュしました。

tomoki-ta opened the pull request:
Helloと表示するJavaファイルを追加しました。

Commit:
コミット練習

Shunya-Yamada commented on Main.java:
```diff
@@ -0,0 +1,5 @@
+public class Main{
```
`Main{`
{の前に半角スペースを空けてください。

tomoki-ta replied:
対応しました。

Shunya-Yamada commented on Main.java:
```diff
@@ -0,0 +1,5 @@
+public class Main{
+    public static void main(String[] args){
```
```suggestion
    public static void main(String[] args) {
```
半角スペースを空けてください。

tomoki-ta replied:
対応しました。

tomoki-ta committed:
半角スペースを追加しました。

mainの後の半角スペースを追加しました。
https://github.com/Shunya-Yamada/TECH-BASE/pull/1#discussion_r1791373600

tomoki-ta committed:
半角スペースを空けました

メソッドの半角スペースを空けました。
https://github.com/Shunya-Yamada/TECH-BASE/pull/1#discussion_r1791376396

Co-authored-by: Shunya-Yamada <79089129+Shunya-Yamada@users.noreply.github.com>

Shunya-Yamada reviewed (approved):
LGTM
